commands/login/index.js

```diff
@@ -32,11 +43,28 @@ program
   .description('Log in and create your user configuration to use the other commands.')
   .action(async () => {
     let api;
-    try {
-      api = await retryP(signIn, isLockedError);
-    } catch (e) {
-      return logError(e);
+    const email = await getEmail();
```

Comment:
as we talked about in person move all the  sso check stuff into the retryable func

Reply:
Okay i made this better.